sros2/api/__init__.py

```diff
@@ -145,17 +145,12 @@ def create_governance_file(path, domain_id):
         f.write("""\
 <?xml version="1.0" encoding="UTF-8"?>
 <dds xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:noNamespaceSchemaLocation="http://www.omg.org/spec/DDS-SECURITY/20160303/omg_shared_ca_governance.xsd">
+    xsi:noNamespaceSchemaLocation="http://www.omg.org/spec/DDS-SECURITY/20170901/omg_shared_ca_governance.xsd">
```

Comment:
Why not use `https` here since the URL is redirected to that one anyway?

Same below.

Reply:
Because the examples in the spec and the examples on OMG website use http.
As there is currently no downside in keeping the http link I figured I'd keep it consistent with the examples.
And who knows, maybe OMG plan ons providing an http hosting and not a redirect in the future ? :crossed_fingers: